shared/teams/add-members-wizard/add-from-where.tsx

```diff
@@ -3,20 +3,39 @@ import * as Kb from '../../common-adapters'
 import * as Styles from '../../styles'
 import * as Container from '../../util/container'
 import * as Types from '../../constants/types/teams'
+import * as Constants from '../../constants/teams'
 import * as TeamsGen from '../../actions/teams-gen'
 import {appendNewTeamBuilder} from '../../actions/typed-routes'
 import {ModalTitle} from '../common'
 
+const Skip = () => {
+  const dispatch = Container.useDispatch()
+  const onSkip = () => dispatch(TeamsGen.createFinishNewTeamWizard())
+  const waiting = Container.useAnyWaiting(Constants.teamCreationWaitingKey)
+
+  if (Styles.isMobile) {
+    return waiting ? (
+      <Kb.ProgressIndicator />
+    ) : (
+      <Kb.Text type="BodyBigLink" onClick={onSkip}>
+        Skip
+      </Kb.Text>
+    )
+  } else {
+    return <Kb.Button mode="Secondary" label="Skip" small={true} onClick={onSkip} waiting={waiting} />
+  }
+}
+
 const AddFromWhere = () => {
   const dispatch = Container.useDispatch()
   const nav = Container.useSafeNavigation()
 
   const teamID = Container.useSelector(s => s.teams.addMembersWizard.teamID)
   const newTeam: boolean = teamID === Types.newTeamWizardTeamID
+  const error = Container.useSelector(s => (newTeam ? s.teams.newTeamWizard.error : undefined))
```

Comment:
This is if you hit "skip" and creating the team fails?

Reply:
yeah. Should I rename to `createTeamError`?

Reply:
sounds good